pkg/generators/models/testdata/cases/validation/expected/model_person_test.go

```diff
@@ -1,31 +0,0 @@
-package generatortest
-
-import (
-	"testing"
-
-	validation "github.com/go-ozzo/ozzo-validation/v4"
-	uuid "github.com/satori/go.uuid"
-	"github.com/stretchr/testify/require"
-)
-
-func TestPerson(t *testing.T) {
-	name := "foo"
-	require.NoError(t, validation.Validate(&Person{
-		Age:            18,
-		Gender:         GenderDefault,
-		Name:           "Foo",
-		Hostname:       "foobar.com",
-		Email:          "foo@localhost.de",
-		Uuid:           uuid.NewV4().String(),
-		Base64:         "Zm9vYmFyCg==",
-		FavoriteColors: []Color{ColorBlue},
-		Ip:             "127.0.0.1",
-		Ipv4:           "127.0.0.1",
-		Ipv6:           "::1",
-		Url:            "https://www.google.com",
-		Address: Address{
-			Street: "foo",
-			Name:   &name,
-		},
-	}))
-}
```

Comment:
@trusch Oh, I see you used my script, you should always check all the staged files carefully, this is not supposed to be removed.

Reply:
Thanks, missed that. Nice script otherwise though ;)